CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+## 2.32.0-beta.0 / 2021-11-16
+
+This beta release introduces the Prometheus Agent, a new mode of operation for
+Prometheus optimized for remote-write only scenarios. In this mode, Prometheus
+does not generate blocks on the local filesystem and is not queryable locally.
+Enable with `--enable-feature=agent`.
+
+Learn more about the Prometheus Agent in our [blog post](https://prometheus.io/blog/2021/11/16/agent/).
+
+* [CHANGE] remote-write: Change default max retry time from 100ms to 5 seconds. #9634
+* [FEATURE] Agent: New mode of operation optimized for remote-write only scenarios, without local storage. Enable with `--enable-feature=agent`. #8785
+* [FEATURE] Promtool: Add `promtool check service-discovery` command. #8970
+* [FEATURE] UI: Add search in metrics dropdown. #9629
+* [FEATURE] Templates: Add parseDuration to template functions. #8817
+* [ENHANCEMENT] Promtool: Improve test output output. #8064
+* [ENHANCEMENT] Promtool: Use kahan summation for better numerical stability. #9588
+* [ENHANCEMENT] Remote-write: reuse memory for marshalling. #9412
+* [ENHANCEMENT] Scrape: Add scrape_body_size_bytes metric. Behind `--enable-feature=extra-scrape-metrics` flag. #9569
+* [ENHANCEMENT] TSDB: Add windows arm64 support. #9703
+* [ENHANCEMENT] TSDB: Optimized MemPostings.EnsureOrder(). #9673
```

Comment:
```suggestion
* [ENHANCEMENT] TSDB: Optimized `MemPostings.EnsureOrder()`. #9673
```

From a user's perspective it could be useful to explain what that means, like saving CPU on read/write operations?

Reply:
it's called when we close the head. cc @codesome what can we write here?